endpoints.html

```diff
@@ -8,7 +8,7 @@
   <p>All responses are sent JSON-encoded. The specific responses (successful ones) are described in the documentation section for each method. See <a href="https://en.wikipedia.org/wiki/Json">https://en.wikipedia.org/wiki/Json</a> for more information on JSON.</p>
   <p>
     All input strings should be UTF-8 encoded, our response is always UTF-8 encoded.
-    Optional JSON properties can be omitted from the request, null values are only allowed for strings.
+    Optional JSON properties can be omitted from the request, avoid providing null values.
```

Comment:
All input strings should be UTF-8 encoded, our response is always encoded conforming the Content-Type http header, which for JSON responses is `charset=utf-8`.
Optional JSON properties should be omitted from the request when not used. Do not provide null values.
For backward compatibility reasons we do include them in the v1 responses but we won't in the v2 responses.